x-pack/plugins/security_solution/public/common/components/events_viewer/index.tsx

```diff
@@ -174,6 +178,11 @@ const makeMapStateToProps = () => {
       query: getGlobalQuerySelector(state),
       sort,
       showCheckboxes,
+      // Used to determine whether the footer should show (since it is hidden if the graph is showing.)
+      graphEventId: /** `getTimeline` actually returns `TimelineModel | undefined` */ (getTimeline(
```

Comment:
i was confused looking at the types in my editor. Basically, `getTimeline` returns `TimelineModel | undefined` but is typed as `TimelineModel`.

Reply:
Not sure if this is working correctly or not. Perhaps it's worth investigating?